### DISCUSSION TITLE
What unit is used for total loss function in lcurve.out ?

### DISCUSSION THREAD

**Discussion by ald3hyde** [deepmodeling/deepmd-kit#3668] [General]
Dear all,

I do not know what units the total RMSE has. How is the RMSE of each component and the total RMSE defined? We would appreciate it if you could show us the formula.

**Comment by njzjz** [comment]
The total loss is unitless. For the formula, please read our software paper.

**Reply by ald3hyde** [reply]
Thank you for replaying.
I thought the prefactors were unitless.
